views/index.ejs

```diff
@@ -54,7 +54,7 @@
                     <li>Body: fat</li>
                 </ul>
                 <p>If you don't have data for some of those fields (eg you don't keep a food log), they will just be blank.</p>
-                <p>For more detail, see the <a href="https://wiki.fitbit.com/display/API/Fitbit+Resource+Access+API">Fitbit API documentation.</a></p>
```

Comment:
I do like keeping this link, because it points specifically to what the app uses. However, I agree that it's a bummer that you have to be logged in to see it. Maybe we should include both of these links?